src/components/seo.js

```diff
@@ -35,6 +35,10 @@ function SEO({ description, lang, meta, title }) {
       title={title}
       titleTemplate={`%s | ${site.siteMetadata.title}`}
       meta={[
+        {
+          name: `title`,
+          content: title,
+        },
```

Comment:
I added this because I saw in some post in the interweb that this meta tag was also useful, nothing too fancy here